src/server/req_rerun.c

```diff
@@ -362,7 +362,7 @@ timeout_rerun_request(struct work_task *pwt)
 		conn_idx = connection_find_actual_index(pjob->ji_rerun_preq->rq_conn);
 	}
 	reply_text(pjob->ji_rerun_preq, PBSE_INTERNAL,
-		"rerunning job has timed out");
+		"rerunning job is still in progress for");
```

Comment:
@mike0042 As per the latest discussion on forum, if we change the error number now it will affect backward compatibility of the command. This is the reason that it is left unchanged.